src/hub/action_response.ts

```diff
@@ -5,20 +5,36 @@ export interface ValidationError {
   message: string
 }
 
-export class ActionResponse {
+export interface Errors {
```

Comment:
nit: calling this interface with a plural name indicates that there should be multiple - this looks like it's only describing a single error.  Should be renamed.

Reply:
I named it "Errors" because we are going to have an array of them. so `errors: Errors[]` still open to changing it to singular but I think plural works here because its an array?

Reply:
using `errors: Error[]` is more clear in my opinion, this means it's an array of Error objects.

Reply:
Done